CONTRIBUTING.md

```diff
@@ -1,4 +1,5 @@
-# Contribution Guide #
+# Contribution Guide
```

Comment:
The situation I am afraid of happening is that you think this way is better while others think another way is better. The result of this would be different group people create a lot redundant PR for just changing the syntax which basically change nothing.